config/management/genesis/src/command.rs

```diff
@@ -290,7 +322,8 @@ pub mod tests {
         let layout_text = "\
             operators = [\"alice\", \"bob\"]\n\
             owners = [\"carol\"]\n\
-            libra_root = [\"dave\"]\n\
+            libra_root = \"dave\"\n\
+            treasury_root = \"other_dave\"\n\
```

Comment:
Interesting, so treasury root is the same account, just a different key?

Reply:
the treasury account is a different account from the libra root account